### PR TITLE
Remove deprecated load methods

### PR DESCRIPTION
**Pull request by kian-thompson**
## Description

`ContainerRuntime.load(...)` and `FluidDataStoreRuntime.load(...)` have been deprecated for some time. Getting rid of these methods will help with the effort to move to the `entryPoint` pattern.

### Removed `ContainerRuntime.load(...)`

The static method `ContainerRuntime.load(...)` has been removed. Please migrate all usage of this method to `ContainerRuntime.loadRuntime(...)`.

### Removed `FluidDataStoreRuntime.load(...)`

The static method `FluidDataStoreRuntime.load(...)` has been removed. Please migrate all usage of this method to `FluidDataStoreRuntime` constructor.

### CODE REVIEW

File: examples/data-objects/codemirror/src/index.ts
Comment (anthony-murphy) [inline examples/data-objects/codemirror/src/index.ts:79]: this is an odd sequencing of changes to me. i think it would be better to first remove all usages, and then remove the methods. i don't like that addition of all the todos.

**Comment by markfields** [comment]
This is hard to review with the other changes baked in.  I'd mark as Draft and wait for review until the first PR merges and flows to `next`.  If you want early feedback you can set up another draft PR _into that branch_ to illustrate the 2ndary changes that will follow.

File: .changeset/fuzzy-cities-wonder.md
Comment (agarwal-navin) [inline .changeset/fuzzy-cities-wonder.md:5]: nit: Removed

File: .changeset/metal-geese-battle.md
Comment (agarwal-navin) [inline .changeset/metal-geese-battle.md:6]: nit: Removed

**Review by agarwal-navin — APPROVED** [review]

**Pull request merged**
